src/families/ethereum/modules/send.js

```diff
@@ -99,13 +106,60 @@ const send: ModeModule = {
     }
   },
 
-  fillDeviceTransactionConfig({ status: { amount } }, fields) {
+  fillDeviceTransactionConfig(
+    { transaction, account, status: { amount } },
+    fields
+  ) {
     if (!amount.isZero()) {
       fields.push({
         type: "amount",
         label: "Amount",
       });
     }
+    if (transaction.data?.length) {
+      const token = findTokenByAddress(transaction.recipient);
+      // $FlowFixMe (transaction data is not null, you flow)
+      const method = transaction.data.slice(0, 4).toString("hex");
+
+      if (method === "095ea7b3" && token) {
```

Comment:
🤔 
to me this code belongs not to send but to `modules/erc20`

the code here feels quite difficult to maintain on long term and we don't want to add one entry for each future feature of ETH.